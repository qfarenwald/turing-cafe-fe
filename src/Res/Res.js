import React from 'react';
import './Res.css';

const Res = ({ id, name, date, time, number }) => {
  return (
    <section>
      <h3>{name}</h3>
      <p>{date}</p>
      <p>{time}</p>
      <p>{number}</p>
      <button>Cancel</button>
    </section>
  )
}

export default Res;
