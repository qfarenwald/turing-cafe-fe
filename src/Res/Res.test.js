import React from 'react';
import ReactDOM from 'react-dom';
import Res from './Res';
import { shallow } from 'enzyme';

describe('Res', () => {
  it('should match snapshot', () => {
    let wrapper = shallow(<Res
      key={1}
      id={1}
      name="Quinne"
      date="12/29"
      time="7:00"
      number={4}
      />)

    expect(wrapper).toMatchSnapshot();
  })
})
