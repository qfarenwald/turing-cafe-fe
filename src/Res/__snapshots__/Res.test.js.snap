// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Res should match snapshot 1`] = `
<section>
  <h3>
    Quinne
  </h3>
  <p>
    12/29
  </p>
  <p>
    7:00
  </p>
  <p>
    4
  </p>
  <button>
    Cancel
  </button>
</section>
`;
